hack/arktos-cni.rc

```diff
@@ -66,6 +66,29 @@ install_cni_bridge() {
     echo "done with bridge cni plugin installation"
 }
 
+check_mizar_prerequisite() {
+    [[ ! -f "/usr/bin/pip3" ]] && sudo apt install python3-pip -y
+    declare -a packages=( netaddr docker grpcio grpcio-tools scapy kubernetes )
+        for package in ${packages[@]}; do
+            if [[ $(pip3 freeze | grep "\b$package") ]]; then
+                continue
+            else
+                sudo wget https://raw.githubusercontent.com/CentaurusInfra/mizar/dev-next/bootstrap.sh -P /tmp/
```

Comment:
This is wrong. You should not run bootstrap script in order to deploy mizar yaml. Bootstrap script does a lot more - it is intended to setup a dev environment for building mizar and running it in kind-setup. All that is not required for running mizar daemon, operator, and CNI. 
Figure out what node-init does not do that you need from bootstrap.

Reply:
We tried this approach as `grpcio` was taking long time while running node-init script and mizar-daemon pod takes long time to come in running state , however this does not resolves the issue and hence we have removed the prior code